easylistpolish/easylistpolish_specific_hide.txt

```diff
@@ -1331,6 +1331,7 @@ allegro.pl##[data-box-name*="advertising"]
 allegro.pl##[data-box-name="Container PPC Offers"]
 allegro.pl##[data-box-name^="Ads"]
 allegro.pl##[data-role="opbox-expandable-advertisement"]
+allegro.pl#?#.mli2_0:-abp-has(a[href^="https://allegro.pl/kategoria/smartfony"])
```

Comment:
Adding a rule that's catching elements by a regular category probably isn't the greatest idea ;) This may brake the website and block users from using a category navigation, which has nothing to do with ads

Reply:
path `kategoria/smarfony` is just a part of allegro.pl service. It is not connected with any ads. If you add this kind of rule you can probably break header or footer of website which may link to this path.

Reply:
Sure thing, my bad! I completely agree. Any suggestions on that ? 
https://allegro.pl/kategoria/smartfony-i-telefony-komorkowe-apple-48978?string=%22iPhone%2014*%22&stan=nowe&offerTypeBuyNow=1&kolor=%C5%BC%C3%B3%C5%82ty

`nowe&offerTypeBuyNow` could be used, although it would remove items from the "Garden related" carousel item.

Reply:
No parts of URL should be used, you should instead look for other attribute selectors